ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.scss

```diff
@@ -198,6 +198,7 @@
       flex-direction: column;
       align-items: center;
       flex: 1;
+      width: 100%;
```

Comment:
This is not a blocker. Just wondering, what prompted you to add this in?

Reply:
With a sufficiently long origin, the section was extending past 100% (beyond the borders of the notification window)

Reply:
Thanks for the explanation! 👍